### DISCUSSION TITLE
Contract 'AdvancedCollectible' object has no attribute 'addresss'

### DISCUSSION THREAD

**Discussion by luahana** [smartcontractkit/full-blockchain-solidity-course-py#1249] [Q&A] [answered]
Hi,

I am on NFT section. approximately 10:52:45 on Video.

I did all the previous steps including deploy it to rinkeby but when I run create_coolectible.py script, I am getting an error saying:

"Contract 'AdvancedCollectible' object has no attribute 'addresss'"

is this because I did not verify my code on rinkeby?

**Comment by cromewar** [comment, marked as answer]
Hello there,
As you can see the error says not attribute "address's" with triple S, look forward your code for a typo regarding for that word.

If that's a typo here and not in the code, please share the complete error message including your code.

Cheers.

**Reply by luahana** [reply]
lol thanks!
